src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+
+// Creates a PostalCode class to take in PostalCode variable
+public class PostalCode {
+
+    private static String post;
```

Comment:
`post` is short, but not a good variable name. BTW, you can use `value`. Also you can implement a method to check if the postal code is in the correct format, e.g. number only.